social-web-maintenance-wg-charter.html

```diff
@@ -170,7 +170,7 @@ <h2>Motivation and Background</h2>
         <h2>Scope</h2>
 
         <p>
-          The Working Group will maintain the
+          Where an active Editor and at least one member unaffiliated with that Editor have volunteered and been approved by the Community Group, the Working Group can maintain any of the deliverables of the previous Social Web Working Group:
```

Comment:
```suggestion
          Where an active Editor and at least one member unaffiliated with that Editor have volunteered and been approved by the Chairs, the Working Group can maintain any of the deliverables of the previous Social Web Working Group:
```
@tantek would this be better? You make a good point that the CG could veto/logjam WG work if they had direct say on which things go into active maintenance, but I had actually been thinking about this in the opposite direction, i.e. WG chairs making CG work (early in the staging process) redundant by, say, expanding the scope of an existing normative work item and "going over their head".  Maybe that last corner-case is not really something we can charter ourselves against, and the best we can require is WG Chairs to keep the CG/list informed when maintenance begins on any items?